contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -617,4 +617,66 @@ GRANT EXECUTE ON PROCEDURE sys.babelfish_remove_domain_mapping_entry TO PUBLIC;
 
 CREATE OR REPLACE PROCEDURE sys.babelfish_truncate_domain_mapping_table()
   AS 'babelfishpg_tsql', 'babelfish_truncate_domain_mapping_table_internal' LANGUAGE C;
-GRANT EXECUTE ON PROCEDURE sys.babelfish_truncate_domain_mapping_table TO PUBLIC;
\ No newline at end of file
+GRANT EXECUTE ON PROCEDURE sys.babelfish_truncate_domain_mapping_table TO PUBLIC;
+
+CREATE TABLE sys.babelfish_extended_properties (
```

Comment:
Please mark this relation as configuration relation using  `pg_extension_config_dump()` function, so that data/content is also dumped along with relation. Every babelfish catalog should be marked as configuration relation.

Reply:
done

Reply:
@markrui3 If we want the data of a particular catalog table to be dumped by `pg_dump` then we would need to mark that table as config table (using `pg_extension_config_dump()`). This is needed to support `pg_dump/pg_dumpall/pg_restore` utilities, `pg_upgrade` works with/without this change. Now, short answer of your question is "Yes, we need to modify PG as well". The reason for this that we have added logic in `pg_dump` such that dump fails if it detects a new unhandled catalog table ([reference1](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/BABEL_3_X_DEV__PG_15_X/src/bin/pg_dump/dump_babel_utils.c#L897) and [reference2](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/BABEL_3_X_DEV__PG_15_X/src/bin/pg_dump/dump_babel_utils.c#L844)), which is because we want to make sure that the developer adding a new catalog considers it's dump/restore impacts so that we don't blindly dump whole contents of the table.  
`Hint`: You can look at the above two references (functions `addFromClauseForLogicalDatabaseDump` and `addFromClauseForPhysicalDatabaseDump`) and add logic to dump the contents of this new catalog table. (Comments around these functions should tell you how to do that)
Finally, to test this, we can add our test file in `test/JDBC/upgrade/master/schedule` file which will be executed in pg_dump/restore github actions.